embedchain/loaders/pdf_file.py

```diff
@@ -16,8 +15,10 @@ def load_data(self, url):
             content = clean_string(content)
             meta_data = page.metadata
             meta_data["url"] = url
-            output.append({
-                "content": content,
-                "meta_data": meta_data,
-            })
-        return output
\ No newline at end of file
+            output.append(
```

Comment:
@sahilyadav902 : can you remove this as its related to this PR?

Reply:
Yeah, I've committed the changes to GitHub directly
My linter was not letting me keep the brackets in the same line

It's fixed now, you may merge